### PR TITLE
Feature/location and tests

### LINKED ISSUE #27: AQ Emphasis
- [x] On Home: 2 horizontal galleries for top cities for AQ-- pollution + ozone (hardcoded?) [Lung Assoc of America]
(https://www.stateoftheair.org/city-rankings/cleanest-cities.html)
Ozone: Anchorage, AK; Savannah, GA; Destin, FL; Honolulu, HI; Casper, WY
Particle Pollution: Hot Springs, AR; Colorado Springs, CO; Bellingham, WA; Albuquerque, NM; Portland, ME;

- [x] On Details: Expand on AQ section and add icons for activity, health, dog 

Data we want:
- general weather (Temp, feels like?, Humidity, Precip, cloudy?, snow, storm, visibility, time) ----> Open Weather API
- aq (pollution, ozone), -text description about weather and aq ---->  AirNow API

What we communicate:
- activities you can do
- optimal places to travel
- health

### LINKED ISSUE #67: Update Air Quality box on Location page
- [x] Add pollen, allergies and wildfire : Coming Soon!
- [x] If we get the wildfire details from air quality API (category.discussion?), display that!

### LINKED ISSUE #75: Add Activity Box on Location
Add an Activity Box with 3 icons and a string that change color based on the UV and AQI value in `Location`

### PR DESCRIPTION
**Pull request by leighlars**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
`ReadMe`, and tsx and tests for `Location`, `About`, `Weekly Forecast` 
We need to add ReadMe gifs, but most of it is done
Tests need to be complete for Location and Weekly Forecast
Slight styling issue on temp in weekly forecast cards
About has disclaimer for air quality data
Styling on Home cards

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
Add ReadMe, style weeklyForecast, add + style Activities + icons to AQI based on UVI or AQI
## Has This Been Tested?
- [X] No
- [X] Yes

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Tests started, didn't finish

# Message for reviewer:
`Your message here`

# Checklist:
- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/About/About.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ const About: React.FC = () => {
       loved ones to have accurate weather and air quality information.
       <br />
       <br />
-      Our data is updated every 10-60 minutes, giving you the most up-to-date
-      conditions to plan your local or destination activity.
+      Our data is updated every 10-60 minutes from local reporting, giving you the most up-to-date
+      conditions to plan your local or destination activity. Note: some stations may not report air quality on a given day.
```
Comment (ErinUntermeyer) [inline src/About/About.tsx:36]: Good idea to add this note that sometimes the data is not reported by stations.

File: src/Location/Location.tsx [hunk 2]
```diff
@@ -90,79 +152,93 @@ const Location: React.FC<LocationProps> = (props) => {
       <div className="info-box air-quality">
        <h4 className="info-box-header">AIR QUALITY</h4>
        <article className="aq-section">
-        <span className='aq-top'>
+        {props.detailsData.currentAir
+         ? activityIcons(
```
Comment (JoshSevy) [inline src/Location/Location.tsx:156]: Ill look into why its throwing a type script error here

File: src/Card/Card.tsx [hunk 3]
```diff
@@ -21,31 +21,37 @@ export interface CardProps {
...
+       {temp(props.temp)}
+       <p className="unit">&deg;F</p>
+      </span>
+      <span className="card-list">
+       {uvIndex(props.uvi)}
+       <p className="unit">UVI</p>
+      </span>
+      {props.aqi || props.aqiCat ? (
+       <span className="card-list">
+        {aqIndex(props.aqi, props.aqiCat)}
```
Comment (ErinUntermeyer) [inline src/Card/Card.tsx:46]: I'm so happy you were able to get this working and displaying! Way to push through!

File: src/Home/Home.tsx [hunk 4]
```diff
@@ -143,28 +143,43 @@ const Home: React.FC<HomeProps> = props => {
...
+					{cardList.slice(10, 15)}
+				</div>
+				<h2 className="carousel-header">Pertinent Readings</h2>
+				<div className="card-carousel">
+					{newsCards}
+				</div>
+				</>
+				) : (
+					<> 
+					<iframe
```
Comment (ErinUntermeyer) [inline src/Home/Home.tsx:172]: Nice added feature!

**Review by JoshSevy — APPROVED** [review]
Looks great, thanks for putting in all this work. These long nights AHHHH

File: src/Location/Location.test.tsx [hunk 5]
```diff
@@ -1,35 +1,237 @@
 import React from "react";
-import { render, getAllByRole} from "@testing-library/react";
+import { render, getAllByRole, getAllByText, getByAltText} from "@testing-library/react";
 import Location from './Location'
+import { MemoryRouter } from "react-router-dom";
+import { mocked } from "ts-jest/utils";
+jest.mock('../helpers/apiCalls.tsx')
 
 describe('Location', () => {
+  // not sure how to test for both the fetched data and the weekly
```
Comment (ErinUntermeyer) [inline src/Location/Location.test.tsx:9]: The full mocked result includes both current and weekly as a single object with two keys (currentWeather and weeklyWeather) so you will just mock the full object one time any time you are testing for current or for weekly.

**Review by ErinUntermeyer — APPROVED** [review]

**Pull request merged**